core/partitioning/shape_analysis.cpp

```diff
@@ -81,8 +81,13 @@ void getSegmentsOutputByRunning(
       jit_inputs_ivalues.push_back(ivalues_maps[input].toList());
     } else if (input->type()->kind() == torch::jit::TypeKind::TupleType) {
       jit_inputs_ivalues.push_back(ivalues_maps[input].toTuple());
+    } else if (input->type()->kind() == torch::jit::TypeKind::NumberType) {
+      jit_inputs_ivalues.push_back(ivalues_maps[input].toScalar());
     } else {
-      TORCHTRT_THROW_ERROR("Unable to find type for value: " << input->debugName() << " to get the ivalues.\n");
+      TORCHTRT_THROW_ERROR(
+          "Unable to find type for value: " << input->debugName()

```

Comment:
This is going to confuse people because they will think why are you throwing an error if you know what the type is supposed to be

Reply:
ok, let me revert this part

Reply:
Maybe just rewording it to say like unexpected type conflict, expected X but saw Y

Reply:
sounds good